ocaml/xapi/helpers.ml

```diff
@@ -1312,3 +1312,7 @@ let retry ~__context ~doc ?(policy = Policy.standard) f =
 
 let retry_with_global_lock ~__context ~doc ?policy f =
   retry ~__context ~doc ?policy (fun () -> with_global_lock f)
+
+let vdi_operation_can_be_performed_live op = match op with
+  | `snapshot | `resize_online | `blocked | `clone | `mirror | `copy -> true
```

Comment:
Are we 100% sure that copy can be performed live? From the code that we moved it looks like (and seems reasonable)  we can copy only if we attached ro (there could be writes during copy that invalidate the copy when you think about it).